test/src/main/java/org/apache/accumulo/test/functional/FateConcurrencyIT.java

```diff
@@ -204,6 +207,8 @@ public void getFateStatus() {
     Instance instance = connector.getInstance();
     String tableId;
 
+    runMultipleCompactions();
```

Comment:
why are we running the multiple compactions here.  I think this should replace the starting of a compaction on line 226 or should not be called at all.

Reply:
Ah, I see that this was disabled by the boolean, and this was only to force the error.  At least add a comment to that effect here.